package.json

```diff
@@ -6,6 +6,8 @@
     "build": "yarn build:quiet --progress",
     "build-prod:quiet": "yarn build:clean && yarn build-photon && cross-env NODE_ENV=production webpack",
     "build-prod": "yarn build-prod:quiet --progress",
+    "build-l10n-prod:quiet": "yarn build:clean && yarn build-photon && cross-env NODE_ENV=production L10N=1 webpack",
+    "build-l10n-prod": "yarn build-l10n-prod:quiet --progress",
```

Comment:
Can you please add a `build-l10n` and `start-l10n` that wouldn't be "prod"? (We don't need a :quiet versions of those IMO).

The reason is that this is faster, and we get an output without uglifying which is easier for debugging.